BootloaderCommonPkg/Library/MmcAccessLib/MmcAccessLib.c

```diff
@@ -303,6 +303,7 @@ EarlyMmcInitialize (
   }
 
   if (Private->Slot.CardType == SdCardType) {
+    MicroSecondDelay (50000);
```

Comment:
This is a common library that could be used by all the platforms, and some platforms don't need this delay. 
Instead of using hard-coded value to delay 50ms, is it possible to use a loop to poll the status or just call SdCardVoltageCheck () until it returns success in given timeout time (50ms)?